addons/mail/controllers/thread.py

```diff
@@ -163,3 +164,35 @@ def mail_message_update_content(self, message_id, body, attachment_ids, attachme
 
     def _is_message_editable(self, message, **kwargs):
         return message.sudo().is_current_user_or_guest_author or request.env.user._is_admin()
+
+    @http.route("/mail/message/get_followers", methods=["POST"], type="json", auth="public")
+    def message_get_followers(self, thread_id, thread_model, limit=20, offset=0, filter_recipients=False):
```

Comment:
Makes more sense to call route `mail/thread/get_followers`: it's to fetch followers of thread. `message` is a different target and multiplicity than thread for followers.

Function name doesn't matter much, but could be `mail_thread_get_followers` for straight-forward and consistent naming with other controller methods.